blocks/components/editable/index.js

```diff
@@ -65,12 +65,13 @@ export default class Editable extends wp.element.Component {
 	}
 
 	onChange() {
-		if ( ! this.editor.isDirty() ) {
-			return;
+		if ( this.props.onChange ) {
+			const value = this.editor.getContent();
+
+			if ( value !== this.props.value ) {
+				this.props.onChange( value );
+			}
 		}
-		const value = this.editor.getContent();
-		this.editor.save();
-		this.props.onChange( value );
```

Comment:
I think relying on the `dirty` status is a good thing. More performant that value checks, especially after a rebase (which will update the shape of the value).

These changes were mandatory for this PR?